docs/index.md

```diff
@@ -1,53 +1,42 @@
 ---
 id: index
-title: "Welcome to Rucio's documentation!"
-sidebar_label: "Welcome!"
+title: "Welcome to Rucio's documentation"
+sidebar_label: "Welcome"
 slug: /
 ---
 
 Rucio is a project that provides services and associated libraries for allowing
 scientific collaborations to manage large volumes of data spread across
 facilities at multiple institutions and organisations. Rucio has been developed
-by [the ATLAS experiment](https://atlas.cern/)
+by [the ATLAS experiment](https://atlas.cern/).
 
 Rucio offers advanced features, is highly scalable, and modular. It is a data
-management solution that could cover the needs of different communities in the
+management solution that covers the needs of different communities in the
 scientific domain (e.g., HEP, astronomy, biology).
 
 Below are some resources to help you get you started on your journey.
 
-## [Before You Get Started](before_you_get_started.md)
+## Getting Started
 
 What exactly is Rucio? What were the motivations behind developing such a
 system? Who uses it? What powers these systems? Answers to all these questions
 and more can be found by browsing through the sub-sections of this topic.
 
-- [What is Rucio?](what_is_rucio.md)
+- [What is Rucio](what_is_rucio.md)
 - [Main Components](main_components.md)
-- [Additional layers and Resources](additional_layers_and_resources.md)
-- [Release Policy](releasepolicy.md)
+- [Additional Layers and Resources](additional_layers_and_resources.md)
 
-## Try Rucio
+## Client
 
-Get your hands dirty by trying to setup your very own demo environment on your
-local laptop/desktop. Make sure you read the pre-requisites section before
-starting off, though!
+The rucio client enables you to access the distributed data. The client can
```

Comment:
`The rucio client enables users to interact with the system and access the distributed data.`